tle/cogs/codeforces.py

```diff
@@ -33,13 +34,28 @@ def __init__(self, bot):
         rc = cf_common.user_db.update_status(active_ids)
         await ctx.send(f'{rc} members active with handle')
 
-    async def _gitgud(self, ctx, handle, problem, delta):
+    async def _validate_gitgud_status(self, ctx, delta):
+        if delta % 100 != 0:
+            await ctx.send('Delta must be a multiple of 100.')
```

Comment:
Instead of printing a message, you should raise an exception containing the error message and it will be caught and an embed with the error message will be displayed. (see bottom of the file)